sites/main-site/src/content/blog/2024/seqera-containers-part-2.mdx

```diff
@@ -0,0 +1,646 @@
+-   [Apptainer](https://apptainer.org/): Alternative to Singularity, uses same image format
+-   [Mamba](https://mamba.readthedocs.io): Alternative to Conda, uses same conda environment files
+
+| `-profile`             | Status                                                 | Use case                                                                           |
+| ---------------------- | ------------------------------------------------------ | ---------------------------------------------------------------------------------- |
+| `docker`               | <span class="badge text-bg-secondary">Unchanged</span> | Docker images for `linux/amd64`                                                    |
+| `singularity`          | <span class="badge text-bg-secondary">Unchanged</span> | Singularity images for `linux/amd64`                                               |
+| `docker_arm`           | <span class="badge text-bg-success">New</span>         | Docker images for `linux/arm64`                                                    |
+| `singularity_arm`      | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/arm64`                                               |
+| `singularity_oras`     | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/amd64` using the `oras://` protocol                  |
+| `singularity_oras_arm` | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/arm64` using the `oras://` protocol                  |
+| `apptainer`            | <span class="badge text-bg-warning">Updated</span>     | Singularity images for `linux/amd64` (not Docker, as previously)                   |
+| `apptainer_arm`        | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/arm64`, using apptainer                              |
+| `apptainer_oras`       | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/amd64` using the `oras://` protocol, using apptainer |
+| `apptainer_oras_arm`   | <span class="badge text-bg-success">New</span>         | Singularity images for `linux/arm64` using the `oras://` protocol, using apptainer |
+| `conda`                | <span class="badge text-bg-warning">Updated</span>     | Conda lockfiles for `linux/amd64`                                                  |
+| `conda_arm`            | <span class="badge text-bg-success">New</span>         | Conda lockfiles for `linux/arm64`                                                  |
+| `conda_env`            | <span class="badge text-bg-success">New</span>         | Conda environments with local environment resolution                               |
+| `mamba`                | <span class="badge text-bg-warning">Updated</span>     | Conda lockfiles for `linux/amd64`, using Mamba                                     |
```

Comment:
What are lock files? This might be something to also introduce wit hteh `arm64` etc thing in the paragraph above

Reply:
There's a collapsed admonition about these further down, but yeah I agree it could do with being moved up / made more obvious.

Reply:
Done in c5b13ffa6